src/lib/dnssd/IncrementalResolve.cpp

```diff
@@ -167,6 +168,7 @@ CHIP_ERROR IncrementalResolver::InitializeParsing(mdns::Minimal::SerializedQName
 
     switch (mServiceNameType)
     {
+    /* All known dnssd records for MATTER are filled into DnssdNodeData */
```

Comment:
What is DnssdNodeData?

Reply:
Seems to have sneaked in during auto merge of rebase. Will correct it.

Reply:
removed the comment.